packages/ui/app/src/api-page/markdown/Markdown.tsx

```diff
@@ -10,6 +10,11 @@ import styles from "./Markdown.module.scss";
 
 export declare namespace Markdown {
     export interface Props {
+        /**
+         * - `"markdown"`: Markdown pages.
+         * - `"api"`: API reference pages.
+         */
+        type: "markdown" | "api";
```

Comment:
I'd probably have made this a bit more declarative (i.e. what is this config actually _doing_? `style="danger"` is better imo than `type="foo"`).

I could see this component staying agnostic to fern (right now it's just a generic markdown component), and utilizing `className` instead. And maybe making a wrapper component to stay DRY.

Reply:
sounds good, yeah def not happy with the `type` solution. was the quickest way to implement the desired functionality (separated font styles for custom markdown and api pages)